### PR TITLE
Update AirAsia.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#9906

### CODE REVIEW

**Comment by J0WI** [comment]
I have no idea what 'broken' means here. The last commit is from 2013, so it might be not up to date.
Would you like to update and re-enable it?

File: src/chrome/content/rules/AirAsia.xml [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+<ruleset name="AirAsia (partial)">
+	<target host="airasia.com" />
+	<target host="www.airasia.com" />
+	<target host="assistive.airasia.com" />
+	<target host="booking.airasia.com" />
+	<target host="booking2.airasia.com" />
+	<target host="giftvoucher.airasia.com" />
+	<target host="member.airasia.com" />
+	<target host="mobile.airasia.com" />
+	<target host="redtix-tickets.airasia.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/AirAsia.xml]:
```
allstarsbooking.airasia.com
aseanpass.airasia.com
cargo.airasia.com
checkin.airasia.com
cms.airasia.com
eform.airasia.com
facebook.airasia.com
hris.airasia.com
itinportal.airasia.com
people.airasia.com
qz8501.airasia.com
redcrew.airasia.com
redemption.airasia.com
redicons.airasia.com
redtix.airasia.com
selfservice.airasia.com
stgmember.airasia.com
stgsfcms.airasia.com
taxinvoice.airasia.com

**Comment by cschanaj** [comment]
@J0WI the domains are added accordingly. thanks!!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**